jcl/j9modules.xml

```diff
@@ -83,6 +91,19 @@
 					</else>
 				</if>
 			</case>
+			<case value="jdk.jconsole">
+				<replaceregexp file="${module.dir}/classes/module-info.java" replace="jdk.management" flags="gs">
+					<regexp pattern="com\.ibm\.management" />
+				</replaceregexp>
```

Comment:
Can we avoid this kind of universal replacement task and use some `macrodef` like `replaceRequires`.